python-eval-mathrepl/mathrepl.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+# File: mathrepl.py
+
+"""MathREPL, a math expression evaluator using Python's eval() and math."""
+
+import math
+
+__version__ = "1.0"
+__author__ = "Leodanis Pozo Ramos"
+
+ALLOWED_NAMES = {
+    name: obj for name, obj in math.__dict__.items() if "__" not in name
```

Comment:
Same comment as on the article, whether this should be `if not name.startswith("__")`. Make sure to keep them in sync